CHANGELOG.adoc

```diff
@@ -31,6 +31,7 @@
 - https://github.com/eclipse-sirius/sirius-components/issues/1203[#1203] [charts] Add support for d3 bars data-structure and bar-chart representation
```

Comment:
- Can you rename `graph.graphqls` into `charts.graphqls`
- Looking again in `Charts.types.ts` I saw some differences with the `graph.grapqls` and `form.graphqls`:
  1. In `Charts.types.ts`, the interface `GQLChart` should not be declared like an interface but like a _type_. Replace the interface declaration with `type GQLChart = GQLBarChart | GQLPieChart` and make `GQLBarChart` and `GQLPieChart` inherit from `GQLRepresentation` to reflect the fact that the type `GQLChart` is a union of types. You may need to do the same for non-GQL types.
  2. In `Charts.types.ts`, move `metadata: GQLRepresentationMetadata` from `interface GQLChart` to `GQLRepresentation`

Reply:
done